CHANGELOG.md

```diff
@@ -7,7 +7,36 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 
 
 ## Unreleased
-[Commits](https://github.com/scalableminds/webknossos/compare/18.08.0...HEAD)
+[Commits](https://github.com/scalableminds/webknossos/compare/18.09.0...HEAD)
+
+### Added
+
+-
+
+### Changed
+
+-
+
+### Fixed
+
+-
+
+### Removed
+
+-
+
+
+## [18.09.0](https://github.com/scalableminds/webknossos/releases/tag/18.09.0) - 2018-08-20
+[Commits](https://github.com/scalableminds/webknossos/compare/18.08.0...18.09.0)
+
+### Highlights
+- The dashboard gallery loads faster [#3036](https://github.com/scalableminds/webknossos/pull/3036) and tracings in the dashboard can show their descriptions [#3035](https://github.com/scalableminds/webknossos/pull/3035).
+- Managing new users got easier through "new inactive users" notifications[#2994](https://github.com/scalableminds/webknossos/pull/2994), and also team managers can activate them now[#3050](https://github.com/scalableminds/webknossos/pull/3050).
```

Comment:
I'd suggest to add a space in front of the PR link :)
#2994 and #3050

Reply:
Thanks! Changed it :-)